monitoring.md

```diff
@@ -0,0 +1,142 @@
+# Monitoring `amppackager` in production
+
+Once you've run `amppackager` server in production, you may want to monitor its
+health and performance, as well as the performance of the underlying requests to the AMP document server. 
+
+The sections below talk about the things you can monitor, and how to do it.
+
+All the command line examples are for Linux OS.
+
+## Monitoring health
+
+To check `amppackager`'s health curl its `/healthz` endpoint:
```

Comment:
health -> health, also should curl be enclosed in back ticks?

Reply:
Done.